src/api/.rubocop.yml

```diff
@@ -82,3 +82,144 @@ Rails/BulkChangeTable:
     - 'db/migrate/20170912140257_change_kiwi_packages_columns_from_big_int_to_int.rb'
     - 'db/migrate/20171107125828_rename_kiwi_preference_types_to_kiwi_preferences.rb'
     - 'db/migrate/20180516074538_explicitly_set_charset.rb'
+
+Style/MethodCallWithArgsParentheses:
+  Enabled: true
+  Exclude:
+    - 'app/views/**/*'
+    - 'db/migrate/**/*'
+    - 'Gemfile*'
+  # the cop doesn't come with a well defined list of good methods, so this
+  # list is the list of offending functions when introducing. Removing functions
+  # from here leads to autocorrecting calls, if you call a new DSL function, add
+  # it here
+  IgnoredMethods:
+    - abort
+    - add_column
+    - add_element
+    - add_foreign_key
+    - add_index
+    - add_reference
+    - after_action
+    - and_return
+    - around
+    - assert
```

Comment:
I still think all this should be upstream and not here: https://github.com/rubocop-hq/rubocop-rspec/issues/667

Reply:
One does not rule out the other. It still makes sense to apply the cop to OBS to understand what is needed or what one would have to address when providing a patch upstream.

Reply:
why? it is something only RSpec related, not OBS related at all... :confused: